### ISSUE TITLE
NPEs from MemberAvatars in search results

### ISSUE THREAD

**Issue by ara4n** [vector-im/vector-web#2014]
Looks like https://github.com/matrix-org/matrix-react-sdk/pull/430 doesn't work, based on:

```
Uncaught TypeError: Cannot read property 'name' of null()	MemberAvatar.js:51
	_getState			@ MemberAvatar.js:51
	MemberAvatar			@ ReactClass.js:668
	_constructComponentWithoutOwner	@ ReactCompositeComponent.js:296
```

and

![image](https://cloud.githubusercontent.com/assets/1294269/18013869/119029c4-6bb8-11e6-93e7-6672d1a3657e.png)

from current /develop

**Comment by ara4n** [comment]
the fix works, but jenkins has failed to deploy it >:(

**Comment by ara4n** [comment]
hopefully fixed by 881739b
